eth/handler.go

```diff
@@ -51,6 +51,10 @@ var (
 	syncChallengeTimeout = 15 * time.Second // Time allowance for a node to reply to the sync progress challenge
 )
 
+// DefaultProtocolVersions are the supported versions of the `eth` protocol (first
+// is primary).
+var DefaultProtocolVersions = []uint{eth.ETH66, eth.ETH65, eth.ETH64}
```

Comment:
This had to be set in the eth package as the protocol has nested packages logic for (eth|snap) protocols.